modules/basics/src/main/java/com/opengamma/strata/basics/location/Country.java

```diff
@@ -256,11 +259,11 @@ public final class Country
    * <p>
    * This contains all the countries that have been defined at the point
    * that the method is called.
-   * 
+   *
    * @return an immutable set containing all registered countries
    */
   public static Set<Country> getAvailableCountries() {
-    return ImmutableSet.copyOf(CACHE.values());
+    return Guavate.concatToSet(COUNTRY_CODES.get().values(), CACHE.values());
```

Comment:
This should really also be sorted, so use `ConcurrentSkipListMap` as the CACHE implementation.

Would be good to have the test ensure that more instances are returned by this method